src/App.tsx

```diff
@@ -23,11 +25,8 @@ export const App: React.FC = () => {
   const uncompletedLength = todos.filter(({ completed }) => !completed).length;
 
   const addTodoToProcesing = (id : number | null) => {
-    if (!id) {
-      setProcessing([]);
-    } else {
-      setProcessing(prev => [...prev, id]);
-    }
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+    !id ? setProcessing([]) : setProcessing(prev => [...prev, id]);
```

Comment:
please don't use tslint disabled!
in this case, you can do like this:
setProcessing(prev => !id ? [] : [...prev, id]);